robot.py

```diff
@@ -16,6 +16,9 @@ class Robot(MagicRobot):
     flipper = Flipper
 
     def createObjects(self):
+        """corresponds motors to talons 
```

Comment:
Can you remove "corresponds motors to talons"?

It seems to be repeating the code